<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>640</width>
      <height>320</height>
     </rect>
    </property>
    <property name="text">
     <string>Video/Image Viewer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Video_Image"/>
    <addaction name="actionOpen_Images_Directory"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionOpen_Label_View"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionVideoPlay"/>
   <addaction name="actionVideoPause"/>
   <addaction name="actionLabelling"/>
  </widget>
  <action name="actionOpen_Video_Image">
   <property name="text">
    <string>Open Video/Image</string>
   </property>
  </action>
  <action name="actionOpen_Images_Directory">
   <property name="text">
    <string>Open Images Directory</string>
   </property>
  </action>
  <action name="actionOpen_Label_View">
   <property name="text">
    <string>Open Label View</string>
   </property>
  </action>
  <action name="actionLabelling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/UI_img/tool_label_plus.png</normaloff>:/UI_img/tool_label_plus.png</iconset>
   </property>
   <property name="text">
    <string>Labelling</string>
   </property>
   <property name="toolTip">
    <string>Labelling Tool</string>
   </property>
  </action>
  <action name="actionVideoPlay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/UI_img/video_play.png</normaloff>:/UI_img/video_play.png</iconset>
   </property>
   <property name="text">
    <string>VideoPlay</string>
   </property>
   <property name="toolTip">
    <string>video play button</string>
   </property>
  </action>
  <action name="actionVideoPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/UI_img/video_pause.png</normaloff>:/UI_img/video_pause.png</iconset>
   </property>
   <property name="text">
    <string>VideoPause</string>
   </property>
   <property name="toolTip">
    <string>video pause button</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
